soh/.clang-format

```diff
@@ -1,23 +1,25 @@
-IndentWidth: 4
-Language: Cpp
-UseTab: Never
-ColumnLimit: 120
-PointerAlignment: Left
-BreakBeforeBraces: Attach
-SpaceAfterCStyleCast: false
-Cpp11BracedListStyle: false
-IndentCaseLabels: true
-BinPackArguments: true
-BinPackParameters: true
 AlignAfterOpenBracket: Align
+AlignConsecutiveBitFields: AcrossComments
+AlignConsecutiveDeclarations: AcrossComments
+AlignConsecutiveMacros: AcrossComments
+AlignEscapedNewlines: Left
 AlignOperands: true
-BreakBeforeTernaryOperators: true
-BreakBeforeBinaryOperators: None
+AlignTrailingComments: true
 AllowShortBlocksOnASingleLine: true
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
 AllowShortCaseLabelsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: false
-AlignEscapedNewlines: Left
-AlignTrailingComments: true
+BinPackArguments: true
+BinPackParameters: true
+BreakBeforeBinaryOperators: None
+BreakBeforeBraces: Attach
+BreakBeforeTernaryOperators: true
+ColumnLimit: 0
```

Comment:
we have some long lines that actually make sense as long lines, i don't want `clang-format` to break those

Reply:
Ah, so this still lets us choose how long lines should be on a case-by-case basis?

Reply:
not case by case as in `clang-format` can have different column limits for different files. setting it to zero just means `clang-format` won't make *any* long lines short automatically.